hydroengine_service/dgds_functions.py

```diff
@@ -250,11 +272,19 @@ def generate_elevation_map(dataset_list=None):
         dataset_list = ELEVATION_DATA.keys()
 
     mosaic_image = mosaic_elevation_datasets(dataset_list)
-
     data_params = DATASETS_VIS["projects/dgds-gee/bathymetry/gebco/2019"]
-    final_image = visualize_elevation(mosaic_image, data_params=data_params)
-    url = _get_gee_url(final_image)
 
+    if min:
```

Comment:
if min_range and if max_range?

Reply:
if min is not None
See comment on line 584 - # min/max values can be zero, should not be None